README.md

```diff
@@ -1,2 +1,66 @@
-# ansible-workstation
+# Ansible Workstation
 Playbooks to setup developer workstations and laptops.
+
+
+## Installation
+
+  1. Ensure Apple's command line tools are installed (`xcode-select --install`).
+  2. [Install Ansible](http://docs.ansible.com/intro_installation.html).
+  3. Clone this repository to your local drive.
+  4. Run `ansible-playbook main.yml --ask-sudo-pass` to run all playbooks
+
+### Running Select Playbooks
+
+To only run select tagged playbooks, use the following command. The `--tags` flag allows you to run any number of tagged playbooks by name. Tag names can be found under `roles:` in `main.yml`.
+
+    ansible-playbook main.yml -i inventory -K --tags "homebrew,github"
```

Comment:
Can we rename this playbook file `mac-desktop.yml`? I'd like to have other playbooks in this repository, too. @khafa100's next task is going to be to create one for the laptops.